### DISCUSSION TITLE
Returning Interface to previous look

### DISCUSSION THREAD

**Discussion by ivoronline** [Kong/insomnia#6916] [Feedback]
With every new update Insomnia is less and less usable for my needs.
I started using Insomnia for the purpose of having cluttered-free view compared to Postman.
And with every new update interface becomes more cluttered and less usable.
I am using Screenshots of Insomnia to create coding tutorials. 
Since these Screenshots are supposed to fit in a word document it is important for me that insomnia window is not too wide so that content can be properly visible.
I am using Request/Response side by side and with every new version total width of the Insomnia Interface has increased to the size that it now becomes unusable for me for the following reasons:

1. You now have to stretch Insomnia window much wider to switch to side by side Request/Response View. Pasting such window into Word Document makes content to small. Could you add control that Request/Response are always side by side and that I can freely widen the window to fit all the content? Instead of being forced too make it much wider then needed for the content to be displayed.

2.  Could you remove vertical left bar that contains just a single Home button. Or to be able to hide it. Isn't this just a waste of horizontal space for just one button? When there is so much unused room at the top.

3. Ability to remove newly added settings "Base Environment" and manage Cookies" that are now always visible cluttering the view with the useless information that I never use. Distracting students from the presented useful data. 

4. Reduce right padding in the Request/Response/Folder Tree Views so that the content doesn't go so fast in the new line when I reduce the width. So that total width of the Windows can be reduced while keeping content on the same line.

In other words, every new feature that is being added either increases the width of the window making it increasingly harder to keep the total width small needed for screenshots to fit in word document in order to present useful data in large enough font. Or new features add new settings that are cluttering the view. Once I use the settings do I really need to look at them all the time? I want to focus on Request/Response data and minimize anything else. I thought that was the prime goal of Insomnia and it was the reason why I moved away from Postman which became unreadable. What has happened? Is there a new development team in Insomnia? Or Insomnia was never meant to be minimalistic and that is just how first versions looked for the lack of functionalities.

Thank you
